### ISSUE TITLE
Sessieoverzicht laat niet het juiste aantal waarnemingen zien.

### ISSUE THREAD

**Issue by LindaPlaterink** [DeepskyLog/DeepskyLog#624]
Er klopt iets niet in het overzicht per sessie. Als ik kijk in het sessie-overzicht, en filter op mijn naam, dan zie ik 2 sessie die geen waarnemingen hebben, bijv 26/09/2016. Maar als ik bij "mijn waarnemingen" kijk, dan heb ik een hele lijst met waarnemingen van die dag. Als ik naar de details van een waarneming kijk, liggen ze binnen in de periode voor de sessie.  Ik gebruik bij het registreren altijd lokale tijd. Kan het hier wat mee te maken hebben? 

![image](https://cloud.githubusercontent.com/assets/18590440/19034507/20365b0a-8964-11e6-968f-664b2ee1ef33.png)

![image](https://cloud.githubusercontent.com/assets/18590440/19034531/3b182c8c-8964-11e6-9f06-9acb8dec38ad.png)

Groeten,

Linda Platerink

**Comment by TomCorstjens** [comment]
Dat kan inderdaad te maken hebben met lokale tijd, want Deepskylog rekent altijd in UT. Dat staat ook zo aangegeven wanneer je een observatie logt. 

Wanneer je de marges van een sessie met 2 uur groter zet, lost het probleem zicht dan op?

**Comment by LindaPlaterink** [comment]
Ja, als ik de sessie 2 uur vroeger laat beginnen, lost dat het op.
Ik weet dat de applicatie alles in UTM logt. Maar ik verwacht dat als ik mijn waarnemingen en mijn sessies in lokale tijd registreer, dat de juiste waarnemingen bij de sessie dan worden gevonden. Dus ik denk dat er iets aan de query veranderd moet worden. Die moet rekening houden met welke instelling de gebruiker werkt en dan omrekenen. Wel wat werk, begrijp ik.

**Comment by WimDeMeester** [comment]
Ik zal proberen om daar binnenkort eens naar te kijken (als ik de tijd vind).

**Comment by WimDeMeester** [comment]
The laravel version of DeepskyLog uses local time for everything, so the problem will probably be solved. Keeping this ticket open to check when the sessions are implemented in the laravel version.
